src/main/java/az/edu/turing/model/dto/BookingDto.java

```diff
@@ -1,4 +1,58 @@
 package az.edu.turing.model.dto;
 
-public class BookingDto {
+import java.io.Serializable;
+import java.time.LocalDateTime;
+
+public class BookingDto implements Serializable {
+    private long bookingId;
+    private long flightId;
+    private String firstName;
+    private String lastName;
```

Comment:
dto fields should be final, so they can be transferred without changes
(remove setters too, constructor will set final variables)